SECURITY.md

```diff
@@ -0,0 +1,9 @@
+# Security Policy
+
+## Reporting a Vulnerability
+
+To report a vulnerability or a security issue please fill the security advisories form
+[here](../../security/advisories/new),
+send an email to security@codeplay.com or contact us using the
+[contact form on our web page](https://codeplay.com/company/contact/?q=Report%20Security%20Issue).
```

Comment:
I say that in the comment above but this link won't be able to be split in GitHub markdown

Reply:
I think it works if we do it inside the text part.